source/rust_verify/src/driver.rs

```diff
@@ -245,10 +245,13 @@ where
 {
     if !build_test_mode {
         if let Some(VerusRoot { path: verusroot, in_vargo }) = find_verusroot() {
-            verifier
-                .args
-                .import
-                .push((format!("vstd"), verusroot.join("vstd.vir").to_str().unwrap().to_string()));
+            {
+                verifier
+                    .args
+                    .import
+                    .lock().unwrap()
+                    .push((format!("vstd"), verusroot.join("vstd.vir").to_str().unwrap().to_string()));
```

Comment:
This doesn't make sense to me - aren't you going to run this many times on the same `Verifier`? Shouldn't this argument be added when constructing the arguments in the first place rather than here? At that point it should be possible to remove the `Mutex` around the imports field.

Reply:
Agreed, the only place where this is written is during command parsing and here. I made the `Args` to be an `Arc` to avoid duplicating them when we are accessing it from the threads. I'll see to add an argument to the parsing function.